test/enforce-else-after-else-if.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @fileoverview Tests for else-after-else-if rule
+ * @author Michelle Pokrass <michellepokrass@gmail.com>
+ */
+
+'use strict';
+
+var Solium = require("solium");
+
+var userConfig = {
+        "rules": {
+                "security/enforce-else-after-else-if": "error"
+        }
+};
+
+describe('[RULE] enforce-else-after-else-if: Rejections', function () {
+        it('should raise an error for an else if with no else after it', function(done) {
+                var code = [
+                        'contract Foo { function bar(n) { if (n > 10) { return n; } else if (n < 10) { return -n; } } }',
+                        'contract Foo { function bar(n) { if (n > 10) { return n; } else if (n < 10) { return -n; } else if (n == 0) { return 1; } } }',
+                        'contract Foo { function bar(n) { if (n > 10) { return n; } else if (n < 10) { return -n; } else { return 100; } } }',
```

Comment:
Please move the 3rd & 4th test (those who pass without any errors and therefore give `errors.length` = 0) to another `describe` section with description `[RULE] enforce-else-after-else-if: Acceptances`